library/src/wasmJsMain/kotlin/ovh/plrapps/mapcompose/ui/state/MapState.wasm.kt

```diff
@@ -0,0 +1,5 @@
+package ovh.plrapps.mapcompose.ui.state
+
+actual fun getProcessorCount(): Int {
+    return 8
```

Comment:
Perhaps adding a comment here (like what you explained in the PR description) would be nice here.